src/main/kotlin/no/nav/amt/deltaker/bff/deltaker/api/model/EndreSluttarsakRequest.kt

```diff
@@ -1,17 +1,12 @@
 package no.nav.amt.deltaker.bff.deltaker.api.model
 
 import no.nav.amt.deltaker.bff.deltaker.api.utils.validerAarsaksBeskrivelse
-import no.nav.amt.deltaker.bff.deltaker.model.Deltaker
 import no.nav.amt.deltaker.bff.deltaker.model.DeltakerEndring
-import no.nav.amt.deltaker.bff.deltaker.model.DeltakerStatus
 
 data class EndreSluttarsakRequest(
     val aarsak: DeltakerEndring.Aarsak,
 ) {
-    fun valider(opprinneligDeltaker: Deltaker) {
+    fun valider() {
```

Comment:
Jeg har ikke sett så nøye på dette så jeg overser sikkert noe, men jeg skjønner ikke helt umiddelbart hvorfor sjekken her er fjernet. Man må vel i det minste ha en avsluttende status for å få en sluttårsak?

Reply:
oj, good catch :D